src/bci_build/package/appcontainers.py

```diff
@@ -807,3 +807,130 @@ def _get_nginx_kwargs(os_version: OsVersion):
     )
     for tomcat_major, os_version in product(_TOMCAT_VERSIONS, ALL_BASE_OS_VERSIONS)
 ]
+
+SPACK_CONTAINERS = [
+    ApplicationStackContainer(
+        name="spack",
+        package_name="spack-image",
+        pretty_name="Spack package manager for supercomputers - build container",
+        os_version=os_version,
+        is_latest=os_version in CAN_BE_LATEST_OS_VERSION,
+        version="%%spack_version%%",
+        version_in_uid=False,
+        replacements_via_service=[
+            Replacement(
+                regex_in_build_description="%%spack_version%%",
+                package_name="spack",
+                parse_version="patch",
+            ),
+            Replacement(
+                regex_in_build_description="%%spack_version%%",
+                file_name="README.md",
+                package_name="spack",
+                parse_version="patch",
+            ),
+        ],
+        package_list=[
+            "spack",
+            "autoconf",
+            "bison",
+            "ccache",
+            "cmake-full",
+            "flex",
+            "libtool",
+            "libzip-devel",
+            "libcurl-devel",
+            "libopenssl-devel",
+            "m4",
+            "makeinfo",
+            "ncurses-devel",
+            "patchelf",
+            "python3-base",
+            "python3-boto3",
+            "tack",
+            "xz-devel",
+        ],
+        entrypoint=["/bin/bash", "/usr/share/spack/docker/entrypoint.bash"],
+        cmd=["interactive-shell"],
+        build_recipe_type=BuildType.DOCKER,
+        env={
+            "SPACK_ROOT": "/usr",
+            "CURRENTLY_BUILDING_DOCKER_IMAGE": "1",
+            "container": "docker",
+        },
+        extra_files={
+            "_constraints": generate_disk_size_constraints(10),
+            "README.md": r"""# Spack %%spack_version%% Container Image
+-
+Spack is a package manager for supercomputers. It provides build recipies
+for > 6k software components and allows to build entire application stacks
+with little to no prerequisits installed.
+-
+## How to use the image
+-
+This image may be used to build and containerize application stacks using
+Spack. The stack will be installed in a base container such as SLE BCI Base.
+To build a containerized application stack, simply create a file `spack.yaml`
+in an empty directory:
+```
+spack:
+  specs:
+  - <application spec>
+
+  container:
+    format: <container_format>
+    images:
+      build: "<path_to_this_image>"
```

Comment:
Is this the "path to this image" as the path on registry.suse.com, where this image will show up?

Reply:
Yes. I can clarify this in the README.